lib/builtin.pm

```diff
@@ -0,0 +1,93 @@
+package builtin;
+
+use strict;
+use warnings;
+
+# All code, including &import, is implemented by always-present functions in
+# the perl interpreter itself.
+# See also `builtin.c` in perl source
+
+1;
+__END__
+
+=head1 NAME
+
+builtin - Perl pragma to import built-in utility functions
+
+=head1 SYNOPSIS
+
+    use builtin qw( true false isbool );
+
+=head1 DESCRIPTION
+
+Perl provides several utility functions in a package called C<builtin::>.
```

Comment:
Minor but the package name is `builtin` whereas `builtin::` could be referred to as the namespace, and it's not only called that but is that!

```suggestion
Perl provides several utility functions in the package namespace C<builtin::>.
```